src/lib/utils.test.ts

```diff
@@ -146,6 +146,84 @@ describe('utils', () => {
         );
       });
     });
+
+    it(`encodes a CompactBytesArray of tuple (bytes4,address,bytes2)`, () => {
```

Comment:
```suggestion
    it('encodes a CompactBytesArray of tuple (bytes4,address,bytes2)', () => {
```

lets use back quote only when needed